profile/README.md

```diff
@@ -0,0 +1,11 @@
+<img src="https://assets.pinata.cloud/pinnie.png" width="100"   />
+
+# Welcome!
+
```

Comment:
We're glad you're here.

In this space, you'll find tools to supercharge your Pinata developer experience. For detailed information, tutorials and plug-and-play code blocks, head over to our [documentation](https://docs.pinata.cloud/).